src/Modal.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
         var closeDialog = function() {
             ReactDOM.unmountComponentAtNode(self.getOrCreateContainer());
 
-            if (props && props.onFinished) props.onFinished.apply(null, arguments);
+            if (props && props.onFinished) props.onFinished.call(null, false);
```

Comment:
`UserSettings.onEmailDialogFinished` is a similar onFinished callback and expects its first argument to be whether the dialog was accepted or dismissed. It should probably have the same bug (as leave room)